src/main/java/org/broadinstitute/hellbender/utils/fasta/CachingIndexedFastaSequenceFile.java

```diff
@@ -159,7 +160,7 @@ public static CachingIndexedFastaSequenceFile checkAndCreate(final File fastaFil
 
         // determine the name for the dict file
         final String fastaExt = fastaFile.getAbsolutePath().endsWith("fa") ? "\\.fa$" : "\\.fasta$";
-        final File dictFile = new File(fastaFile.getAbsolutePath().replaceAll(fastaExt, ".dict"));
+        final File dictFile = new File(fastaFile.getAbsolutePath().replaceAll(fastaExt, IOUtil.DICT_FILE_EXTENSION));
```

Comment:
Is replaceAll really what you want?  Don't you want to just replace the last one?